files/02_amend_replication_agreements_and_reinitialize.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# An array of valid topology suffixes
+declare -a TOPOLOGY_SUFFIXES=("domain" "ca")
+
+# Enable replication of the krblastsuccessfulauth timestamps on all
+# existing topology segments.  By default these timestamps *are not*
+# replicated between servers to avoid a replication storm, but given
+# our limited number of users we should be OK.  See here for more
+# details: https://pagure.io/freeipa/issue/9821
+function enable_last_successful_auth_replication {
+  for suffix in "${TOPOLOGY_SUFFIXES[@]}"; do
+    # Get all the topology segments for the given topology suffix.  We
+    # will need to update each one.
+    topology_segments=$(ipa topologysegment-find "$suffix" --pkey-only \
+      | sed --quiet "s/^[[:blank:]]*Segment name:[[:blank:]]*\(.*\)$/\1/p")
+    for segment in $topology_segments; do
+      # Get detailed information about the topology segment so we can
+      # modify it.
+      cmd_output=$(ipa topologysegment-find "$suffix" --all \
+        --name="$segment" --raw)
+      # Extract the part where the fractional replication attributes
+      # are specified
```

Comment:
Consistency with periods.

```suggestion
      # are specified.
```